### PR TITLE
Add pr template

### PR DESCRIPTION
**Pull request by aspitz1**
What does this PR do?
* Adds PR template
* [Link to docs on PR templates on GH](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository)

How should this PR be tested?
* Read through the template
* Review for changes
* Feel free to request changes inline on PR

Anything else the reviewer should know?
* I am unsure how to use this in the PR still, so it will be a bit of an experiment

### CODE REVIEW

**Review by tfettig22 — APPROVED** [review]
Looks great!

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+What does this PR do?
+* Please make a bulleted list
+
+How should this PR be tested?
```
Comment (edjrodriguez) [inline pull_request_template.md]:
```suggestion
Where should the reviewer start?
```

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+* Please make a bulleted list
+
+Anything else the reviewer need to know?
```
Comment (edjrodriguez) [inline pull_request_template.md]:
```suggestion
Anything else the reviewer needs to know (e.g. issues, concerns)?
```

**Review by edjrodriguez — COMMENTED** [review]
Love this template

**Pull request merged**